Sources_v3/Controllers/ChannelController/ChannelController.swift

```diff
@@ -116,7 +116,7 @@ public class _ChatChannelController<ExtraData: ExtraDataTypes>: DataController,
     @Atomic public private(set) var channelQuery: ChannelQuery<ExtraData>
 
     /// The identifier of a channel this controller observes.
-    private var channelId: ChannelId { channelQuery.cid }
+    public var cid: ChannelId { channelQuery.cid }
```

Comment:
Why when the value is already available by `controller.channelQuery.cid`?

Reply:
I don't have strong points on that. 
Just decided that it will be more friendly API if we will expose it on controller.